rules/windows/c2_grey_list_beacon_sequence.toml

```diff
@@ -0,0 +1,88 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network activity from unexpected system applications. This may indicate adversarial activity as these
+applications are often leveraged by adversaries to execute code and evade detection.
+"""
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual Process Network Connection: Sequence"
+risk_score = 21
+rule_id = "1fe3b299-fbb5-4657-a937-1d746f2c711a"
+severity = "medium"
+tags = ["from-eql-rules"]
```

Comment:
All of these show up in the UI, so we should keep that in mind. `from-eql-rules` isn't as user friendly as `AWS` or `Identity Monitoring` for example

Reply:
it was just a temporary place holder for me to initially identify 🙂 - already replaced